src/lib/operators/aggregate_hash.hpp

```diff
@@ -69,7 +69,7 @@ using AggregateResultIdMapAllocator = PolymorphicAllocator<std::pair<const Aggre
 template <typename AggregateKey>
 using AggregateResultIdMap =
     ska::bytell_hash_map<AggregateKey, AggregateResultId, std::hash<AggregateKey>, std::equal_to<AggregateKey>,
-                       AggregateResultIdMapAllocator<AggregateKey>>;
+                         AggregateResultIdMapAllocator<AggregateKey>>;
```

Comment:
Is it a bad thing when the formatter changes files that I did not touch?

Reply:
Generally no, unless it adds tens of lines to your PR.